go.mod

```diff
@@ -31,11 +30,11 @@ require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
-	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
-	google.golang.org/grpc v1.23.1
+	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553
+	google.golang.org/grpc v1.26.0
 	k8s.io/api v0.17.0
 	k8s.io/apiextensions-apiserver v0.17.0
-	k8s.io/apimachinery v0.17.0
+	k8s.io/apimachinery v0.17.1-beta.0
```

Comment:
Was this intentional?

Reply:
Nope, it was not.

Reply:
Ok, after removing and re-adding the `go.mod` and `go.sum` files, this is because of `external-snapshotter`'s dependency. https://github.com/kubernetes-csi/external-snapshotter/blob/master/go.mod#L15